lib/Cro/WebApp/Template/AST.pm6

```diff
@@ -318,11 +325,21 @@ my constant %escapes = %(
     "'" => '&apos;',
 );
 
-sub escape-text(Str() $text) {
+multi escape-text(Nil $text, $file, $line) {
```

Comment:
I think we'll want a candidate for `Mu:U` as well, to catch other kinds of type objects.